practice/jokes/jokes/models.py

```diff
@@ -0,0 +1,54 @@
+"""Jokes models."""
+
+
+class Joke:
+    """
+    Value type that represents a submitted joke. It has a setup, and a punchline.
+
+    >>> a = Joke('words', 'wordswords')
+    >>> a
+    'Joke(words, wordswords)'
+    """
```

Comment:
Instead of doing a class-level doctest, do individual function doctests. This allows someone to more easily pinpoint where functionality is incorrect: when a single test fails, you look at that function.